src/features/content-explorer/messages.js

```diff
@@ -82,6 +82,17 @@ const messages = defineMessages({
         description: 'Text shown to indicate the number of items selected',
         id: 'boxui.contentExplorer.numSelected',
     },
+    filesAndFoldersSelected: {
+        defaultMessage: '{numSelected} of folder and files selected',
+        description: 'Text shown to indicate the number of items selected with Include Subfolders feature',
+        id: 'boxui.contentExplorer.filesAndFoldersSelected',
+    },
+    fileSelectedToIncludeSubfolders: {
+        defaultMessage: '{numSelected} folder and all subfolders within is selected',
```

Comment:
if `numSelected` is more than one than we need a plural version of `folder` right?